---
course_id: 7-340-nano-life-an-introduction-to-virus-structure-and-assembly-fall-2005
layout: course_section
menu:
  leftnav:
    identifier: 1fd73e5124b878375b2962e10c63ae31
    name: Readings
    weight: 30
title: Readings
type: course
uid: 1fd73e5124b878375b2962e10c63ae31

---

| WEEKÂ # | TOPICS | OVERVIEW | READINGS |
| --- | --- | --- | --- |
|  {{< br >}}{{< br >}} 1 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} An Introduction to Virus Structure and Assembly {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} During the first class session, the instructors and the students will introduce themselves. The instructors will introduce the course and go over requirements and expectations outlined in the syllabus. The instructors then will present a brief lecture outlining the background and basics of virus structure and assembly. Finally, the topic of week 2 will be introduced. {{< br >}}{{< br >}}  |  |
|  {{< br >}}{{< br >}} 2 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Principles of Virus Symmetry {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Watson and Crick noticed that viral nucleic acid couldn't encode a single protein big enough to enclose a virus and suggested that multiple, identical subunits polymerized into a structure to house viral nucleic acid. Caspar and Klug proposed a geometric formalism, called quasi-equivalence, to explain how 1) a spherical capsid can be built from pentamers and hexamers and 2) a protein can participate in non-equivalent interactions. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Caspar, D. L., and A. Klug. "Physical principles in the construction of regular viruses." _Cold Spring Harb Symp Quant Biol_ 27 (1962): 1-24. {{< br >}}{{< br >}} Liddington, R. C., Y. Yan, J. Moulai, R. Sahli, T. L. Benjamin, and S. C. Harrison. "Structure of simian virus 40 at 3.8-A resolution." _Nature_ 354 (1991): 278-84. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 3 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Virus Crystallography and Cryo-EM {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Crystallography, and more recently, cryoelectron microscopy, are techniques used to determine the structures of viruses. At the time that Wikoff and co-workers solved the structure of a whole virus, HK97, it was the largest structure ever solved by crystallography. Cryo-EM, though not as high resolution, provides information from frozen hydrated specimens without the need for crystals. Wu et al. determined the structure of a herpes virus and showed the locations of the subunits within the structure. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Wikoff, W. R., L. Liljas, R. L. Duda, H. Tsuruta, R. W. Hendrix, and J. E. Johnson. "Topologically linked protein rings in the bacteriophage HK97 capsid." _Science_ 289 (2000): 2129-33. {{< br >}}{{< br >}} Wu, L., P. Lo, X. Yu, J. K. Stoops, B. Forghani, and Z. H. Zhou. "Three-dimensional structure of the human herpesvirus 8 capsid." _J Virol_ 74 (2000): 9646-54. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 4 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Capsid Assembly {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Capsids must be built from their component pieces. They form without any external help or energy in a process often called "self-assembly." Prevelige and King developed one of the earliest in vitro assembly systems for an icosahedral virus. Conway and co-workers used cryo-em to study the conformations of virus capsids as the matured over time, a process that results in the expansion and increased rigidity of the viral shell. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Conway, J. F., R. L. Duda, N. Cheng, R. W. Hendrix, and A. C. Steven. "Proteolytic and conformational control of virus capsid maturation: the bacteriophage HK97 system." _J Mol Biol_ 253 (1995): 86-99. {{< br >}}{{< br >}} Prevelige, P. E., Jr., D. Thomas, and J. A. King. "Nucleation and growth phases in the polymerization of coat and scaffolding subunits into icosahedral procapsid shells." _Biophys J_ 64, no. 3 (1993): 824-35. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 5 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Capsid Assembly II {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Extending the principles described by Caspar and Klug, Ganser et al. propose a fullerene cone model for the HIV core. They later prove their model using an advanced cryo-EM technique called cryotomography. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Ganser, B. K., S. Li, V. Y. Klishko, J. T. Finch, and W. I. Sundquist. "Assembly and analysis of conical models for the HIV-1 core." _Science_ 283 (1999): 80-3. {{< br >}}{{< br >}} Benjamin, J., B. K. Ganser-Pornillos, W. F. Tivol, W. I. Sundquist, and G. J. Jensen. "Three-dimensional structure of HIV-1 virus-like particles by electron cryotomography." _J Mol Biol_ 346 (2005): 577-88. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 6 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Nucleic Acid Packaging {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Viruses have evolved different strategies for getting their nucleic acids into shells. Bacteriophages typically use a preformed protein shell, called a procapsid, into which the DNA genome is pumped by a complex of proteins called portal and terminase. Using optical tweezers, Smith et al. show that one such packaging motor is incredibly strong. Retroviruses have entirely different mechanisms based on interactions between RNA secondary structure and the proteins that make up the surrounding shell. These protein-nucleic acid interactions are an essential component of the simultaneous assembly and packaging of a retrovirus and its genome. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} D'Souza, V., and M. F. Summers. "Structural basis for packaging the dimeric genome of Moloney murine leukaemia virus." _Nature_ 431, no. 7008 (2004): 586-90. {{< br >}}{{< br >}} Smith, D. E., S. J. Tans, S. B. Smith, S. Grimes, D. L. Anderson, and C. Bustamante. "The bacteriophage straight phi29 portal motor can package DNA against a large internal force." _Nature_ 413 (2001): 748-52. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 7 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Virus Recognition and Attachment {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} The first step in a viral infection is the recognition of and attachment to a host cell. The bacteriophage P22 uses a tailspike protein that specifically recognizes and cleaves lipopolysaccharide on the surface of its host, Salmonella typhimurium, enabling it to "burrow" to the cell surface. Steinbacher and collegues determined the structure of the protein in complex with its substrate, revealing the molecular basis for the specificity. Attachment by the dengue viruses is by an equally elegant method. Unlike tailspike, the proteins that accomplish this task undergo major structural rearrangement upon interacting with the host. This restructuring of the protein triggers membrane fusion, a critical step in the infection process. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Modis, Y., S. Ogata, D. Clements, and S. C. Harrison. "Structure of the dengue virus envelope protein after membrane fusion." _Nature_ 427, no. 6972 (2004): 313-9. {{< br >}}{{< br >}} Steinbacher, S., U. Baxa, S. Miller, A. Weintraub, R. Seckler, and R. Huber. "Crystal structure of phage P22 tailspike protein complexed with Salmonella sp. O-antigen receptors." _Proc Natl Acad Sci U S A_ 93, no. 20 (1996): 10584-8. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 8 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Virus Entry {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Once viruses attach to their hosts, they must get their nucleic acid inside the cell. Again, viruses have evolved very diverse solutions to this problem. Poliovirus, a picornavirus, uses a peptide that binds to membranes and induces the cell to internalize the particle through the endocytotic pathway. Bacteriophage T4 instead creates a channel across its host's membranes and releases its DNA into the cell. Petr Leiman and coworkers used cryo-EM to reveal the remarkable structural changes that accompany the protein complexes of the T4 tail that accomplish this feat. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Bubeck, D., D. J. Filman, N. Cheng, A. C. Steven, J. M. Hogle, and D. M. Belnap. "The structure of the poliovirus 135S cell entry intermediate at 10-angstrom resolution reveals the location of an externalized polypeptide that binds to membranes." _J Virol_ 79, no. 12 (2005): 7745-55. {{< br >}}{{< br >}} Leiman, P. G., P. R. Chipman, V. A. Kostyuchenko, V. V. Mesyanzhinov, and M. G. Rossmann. "Three-dimensional rearrangement of proteins in the tail of bacteriophage T4 on infection of its host." _Cell_ 118, no. 4 (2004): 419-29. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 9 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Virus Structures in the Cell {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Viruses are molecular freeloaders and will utilize whatever cell pathways they can. T4 capsid protein requires a chaperone to fold, but the host chaperone is too small. How does T4 deal with this problem? It has a gene encoding a chaperone cap that interacts with the host chaperone, creating a larger space in which to fit the T4 capsid protein for folding. African Swine Fever Virus uses a poorly understood cellular mechanism to assist in the folding and assembly of its capsids: the aggresome. Believed to be a cellular mechanism for dealing with misfolded proteins, these intracellular structures are induced during ASFV infection and are the sites of capsid assembly. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Bakkes, P. J., B. W. Faber, H. van Heerikhuizen, and S. M. van der Vies. "The T4-encoded cochaperonin, gp31, has unique properties that explain its requirement for the folding of the T4 major capsid protein." _Proc Natl Acad Sci U S A_ 102, no. 23 (2005): 8144-9. {{< br >}}{{< br >}} Heath, C. M., M. Windsor, and T. Wileman. "Aggresomes resemble sites specialized for virus assembly." _J Cell Biol_ 153, no. 3 (2001): 449-55. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 10 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Virus Exit and Release {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Once a virus assembles inside a cell, it has to get out. Bacteriophages have a simple but robust system for lysing their hosts, literally causing them to explode. This system has a built-in control mechanism enabling effective timing of lysis. HIV seems to exploit a previously poorly understood cellular pathway for budding from cells. Von Schwedler et al. use genetics to identify over twenty protein players in this very complex pathway. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Grundling, A., D. L. Smith, U. Blasi, and R. Young. "Dimerization between the holin and holin inhibitor of phage lambda." _J Bacteriol_ 182, no. 21 (2000): 6075-81. {{< br >}}{{< br >}} von Schwedler, U. K., M. Stuchell, B. Muller, D. M. Ward, H. Y. Chung, E. Morita, H. E. Wang, T. Davis, G. P. He, D. M. Cimbora, A. Scott, H. G. Krausslich, J. Kaplan, S. G. Morham, and W. I. Sundquist. "The protein network of HIV budding." _Cell_ 114, no. 6 (2003): 701-13. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 11 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Human Immunodeficiency Virus {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Putting together what we have learned about virus structure and assembly, we will discuss HIV antiviral compounds in light of their mechanisms. Viral fusion inhibitors were once a promising anti-HIV therapy. But HIV quickly evolved resistance, and the basis of this resistance is understood in terms of the structure of the protein that mutated, gp41. Sticht et al. report a promising anti-viral peptide that inhibits the assembly of the HIV capsid. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Baldwin, C. E., R. W. Sanders, Y. Deng, S. Jurriaans, J. M. Lange, M. Lu, and B. Berkhout. "Emergence of a drug-dependent human immunodeficiency virus type 1 variant during therapy with the T20 fusion inhibitor." _J Virol_ 78, no. 22 (2004): 12428-37. {{< br >}}{{< br >}} Sticht, J., M. Humbert, S. Findlow, J. Bodem, B. Muller, U. Dietrich, J. Werner, and H. G. Krausslich. "A peptide inhibitor of HIV-1 assembly in vitro." _Nat Struct Mol Biol_ 12, no. 8 (2005): 671-7. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 12 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Project Day: Looking at Virus Structures in 3D {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} The visual manipulation of biomacromolecules in 3D is an important tool for structural biologists. We will spend this class period becoming familiar with research-level software that allows us to rotate, manipulate, and view molecules in 3-dimensions, and we will take a molecular-level look at some of the viral proteins discussed during the semester. {{< br >}}{{< br >}}  |  |
|  {{< br >}}{{< br >}} 13 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Virus Nanotechnology {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Viruses are essentially biological machines. The genetic control of their structures, coupled to the fidelity with which these structures self-assemble, make viruses ideal platforms for the development of nano-scale materials and surfaces. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Mao, C., D. J. Solis, B. D. Reiss, S. T. Kottmann, R. Y. Sweeney, A. Hayhurst, G. Georgiou, B. Iverson, and A. M. Belcher. "Virus-based toolkit for the directed synthesis of magnetic and semiconducting nanowires." _Science_ 303, no. 5655 (2004): 213-7. {{< br >}}{{< br >}} Meunier, S., E. Strable, and M. G. Finn. "Crosslinking of and coupling to viral capsid proteins by tyrosine oxidation." _Chem Biol_ 11, no. 3 (2004): 319-26. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 14 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Field Trip to the Transmission Electron Microscope {{< br >}}{{< br >}}  |  |  |
|  {{< br >}}{{< br >}} 15 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Proposal Presentations {{< br >}}{{< br >}}  |  |